contribute.md

```diff
@@ -0,0 +1,46 @@
+Contributing
+
+Contributions are always welcome, no matter how large or small! Before contributing, please read the code of conduct.
+
+Not sure where to start?
+When you feel ready to jump into the source code, a good place to start is to look for issues tagged with help wanted and/or good first issue.
+
+Setup
+install python v3
+install pip
+intall tensorflow v2
+
+setup pytest as test runner in your IDE
+setup crlf in git config
+
+
+How to run tests
+execute from command line command pytest
+
+Pull Request Checklist
+
+Fork the repository to your GitHub Account.
+Clone forked repo to your local machine
+Create your feature branch
+Make code/documantation changes
+Create pull request
+
+Before sending your pull requests, make sure you followed this list.
+
+Read contributing guidelines.
+Read Code of Conduct.
+Ensure you have signed the Contributor License Agreement (CLA).
```

Comment:
Again, due to a lack of my education in the programming of open source software, what is Contributor License Agreement, and where should it be placed?

Reply:
Contributor License Agreement(CLA) is optional.It is a document resplendent with intellectual
property implications and therefore, a controversial matter for some free and open source software practitioners.
If you have no special requirements to contributors in area of intellectual property implications, you should not create CLA.
CLA has administrative overhead and slowdown in contributions.
Can be placed as text file anywhere in project. You can guarantee agreement by git check - contributor signing their contribution(commit) using the -s or --signoff flags of the git version control system.